modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -54,7 +54,7 @@ def submit
       end
 
       def submit_supporting_documents
-        if %w[40-0247 10-10D 40-10007].include?(params[:form_id])
+        if %w[40-0247 20-10207 10-10D 40-10007].include?(params[:form_id])
```

Comment:
just curious why do we need to specify forms this? it would be nice if we didn't have to go into this function every time and update it

Reply:
Totally agreed! This is one spot I intend to refactor in the future. I'd rather just call the method on all forms and make them all responsive to that method call, resulting in a passthrough `nil` in cases where we don't need supporting documents.